services/listingFunctions.js

```diff
@@ -99,12 +99,13 @@ async function createListing(cityIds, payload, userId, roleId) {
     } else if (payload.description.length > 65535) {
         throw new AppError(`Length of Description cannot exceed 65535 characters`, 400);
     } else {
-        
+
         insertionData.description = checkDesc(payload.description);
     }
     if (payload.media) {
         insertionData.media = payload.media;
     }
+    let isPoll = false;
```

Comment:
Use a valriable called categoryId and store it here. in the if block check if catgoryid === categories.Polls